Tests/PointFreeTests/__Snapshots__/StripeWebhooksTests/testNoInvoiceSubscriptionId_AndNoLineItemSubscriptionId.1.Conn.txt

```diff
@@ -0,0 +1,17 @@
+POST http://localhost:8080/webhooks/stripe
+Cookie: pf_session={}
+Stripe-Signature: t=1517356800,v1=950becadb9b19c86003ad07e745cc0b99a50f76b362bb50d5fe159c1f84c606f
+
+{"id":"evt_test","data":{"object":{"id":"in_test","amount_paid":1700,"customer":"cus_test","lines":{"has_more":false,"data":[{"amount":1700,"id":"ii_test","quantity":1,"plan":{"amount":1700,"name":"Individual Monthly","id":"monthly-2019","nickname":"Individual Monthly","created":1517356800,"tiers":[{"amount":1600},{"amount":1800}],"currency":"usd","metadata":{},"interval":"month"}}]},"charge":"ch_test","total":1700,"period_start":1517356800,"subtotal":1700,"amount_remaining":0,"date":1517356800,"period_end":1519948800,"number":"0000000-0000"}},"type":"invoice.payment_failed"}
+
+400 Bad Request
```

Comment:
if the subscription id is missing from the invoice and all of its line items, we will then return a 400